content/contributing/opencue/getting-started-windows.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Development Environment setup on Windows"
+linkTitle: "Development Environment setup on Windows"
+date: 2020-01-29
+weight: 2
+description: >
+  A guide to setting up an end-to-end development environment on Windows.
+---
+
+<!---DO NOT MODIFY THIS LINE--->
+
+This is a guide to setting up a development environment on Windows that will allow 
+you to run the entire OpenCue system from end-to-end (Cuebot, Cuesubmit, CueGUI, RQD)
+and make changes to any part of it.
+
+## Prerequisites
+
+First, clone the [OpenCue git repository](https://github.com/AcademySoftwareFoundation/OpenCue)
+to your machine. You will then need to download & install three core dependencies:
+
+- [Postgres](https://www.postgresql.org/download/windows/)
+- [Python 3.x](https://www.python.org/downloads/) 
+- [Java SE JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
+
+It is also useful to have IDEs installed. JetBrains has free versions of both:
+
+- [PyCharm](https://www.jetbrains.com/pycharm/)
+- [IntelliJ IDEA](https://www.jetbrains.com/idea/)
+
+## Database setup
+
+After installing Postgres you will need to create a database and user for OpenCue
+(specifically CueBot) to use.
+
+1. **Start pgAdmin**: Open pgAdmin from the Start menu. This will add an icon to the system tray. 
+   Right-click the pgAdmin icon and select “New pgAdmin window…”, which will open 
+   in your browser. Sign in with your Postgres admin user credentials that you selected during
+   Postgres installation.
+1. **Create User**: From the tree-view on the left, find “Login/Group Roles”. Right-click it and select
+   “Create→Login/Group Role…”. Name the user `opencue` and choose a secure password. Under
+   the “Privileges” tab, enable “Can login?”, then hit “💾 Save”.
+1. **Create Database**: From the tree-view on the left, find “Databases”. Right-click it and
+   select “Create→Database…”. Name the database `opencue` and select the `opencue` user as
+   the owner, then hit “💾 Save”.
```

Comment:
The icons are a nice touch, but we don't typically include these as they add to the maintenance overhead of the docs but add little benefit. I can clean these up after you merge the PR, so no need to worry about it.

Reply:
I’ve removed the icons for `💾 Save`, however the reason I originally added them is that pgAdmin has icon-only buttons for the actions in the Query Editor. The two actions necessary are highlighted below:

![Screenshot of pgAdmin query editor interface](https://user-images.githubusercontent.com/12575/73486104-4ae2e680-4409-11ea-99a2-9adb2b8dc70b.png)